src/PrestaShopBundle/Resources/views/Admin/Product/catalog.html.twig

```diff
@@ -27,10 +27,10 @@
 
 {% block javascripts %}
   {{ parent() }}
-  <script src="{{ asset('themes/default/js/bundle/product/catalog.js') }}"></script>
+  {# <script src="{{ asset('themes/default/js/bundle/product/catalog.js') }}"></script> #}
   <script src="{{ asset('themes/default/js/bundle/pagination.js') }}"></script>
   <script src="{{ asset('themes/default/js/bundle/category-tree.js') }}"></script>
-  <script src="{{ asset('../js/jquery/ui/jquery.ui.sortable.min.js') }}"></script>
+  {# <script src="{{ asset('../js/jquery/ui/jquery.ui.sortable.min.js') }}"></script> #}
```

Comment:
Maybe this ?

Reply:
I'll uncomment this in a dedicated PR for the catalog page